ui/i18n/messages.json

```diff
@@ -242,6 +244,7 @@
   "operationButton.bulk.poll.error": "Error while executing {0}",
   "operationButton.bulk.poll.running": "{0} is running ({1}/{2})",
   "operationButton.bulk.poll.scheduled": "{0} is scheduled. You will be notified when it starts.",
+  "parentInspectorButton.tooltip": "Parent Inspector",
```

Comment:
I see the label in this repository, but I don't see where it is used. If the element is on web ui, and the label is only used there, then it should be on the nuxeo web ui repository as well.